pydicom/pixel_data_handlers/pillow_handler.py

```diff
@@ -150,13 +150,17 @@ def get_pixeldata(ds):
     if getattr(ds, 'NumberOfFrames', 1) > 1:
         # multiple compressed frames
         for frame in decode_data_sequence(ds.PixelData):
-            decompressed_image = Image.open(io.BytesIO(frame))
-            pixel_bytes.extend(decompressed_image.tobytes())
+            im = Image.open(io.BytesIO(frame))
+            if 'YBR' in ds.PhotometricInterpretation:
+                im.draft('YCbCr', (ds.Rows, ds.Columns))
```

Comment:
good idea!

Reply:
Let me add a test to confirm this is the right shape

Reply:
Hmm, I don't think it matters anyway since we end up as 1D and reshape...